frontend/src/context/BaseSearchContext.tsx

```diff
@@ -19,33 +21,23 @@ export function normalizeParams<T>(params: object) {
 	)
 }
 
-export type BaseSearchContext<T> = {
```

Comment:
removed a few of the setters that were updating internal state - e.g. `backendSearchQuery` is now updated whenever `setSearchQuery` is called to avoid them being out of sync